### ISSUE TITLE
Running into FFT error, any advice? 

### ISSUE THREAD

**Issue by JustinJaniszewski** [ubsuny/23-Homework5G4#14]
![image](https://github.com/ubsuny/23-Homework5G4/assets/143819849/317556f3-ee07-49d1-b1cd-de997be97825)
not sure what to make of this. any help is appreciated.

**Comment by JustinJaniszewski** [comment]
Cutting the data into a length of 2^n seemed to avoid the problem. I am not sure if that was the point of the error message or not so I will leave the issue open for someone to answer.

**Comment by poojashresthacode** [comment]
Have you resolved this issue?

**Comment by JustinJaniszewski** [comment]
Yes, sort of. I found a way to avoid this problem so I suppose its resolved. However, I am not sure what the cause of the issue was in the first place. I would like to leave this issue open in case somebody from another group can answer it.
